denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/mod_main.dsc

```diff
@@ -11,21 +11,23 @@
 mod_command:
   type: command
   debug: false
-  permission: mod.staff
+  permission: adriftus.staff
   name: mod
   description: Adriftus Moderator Panel
-  usage: /mod (username)
+  usage: /mod [username]
```

Comment:
I think this argument was supposed to be optional, it opens a GUI if no arguments are specified